### PR TITLE
[Behat] Clean up test parameters

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

Clean up test parameters.

### CODE REVIEW

File: etc/travis/run-behat [hunk 1]
```diff
@@ -1,28 +1,11 @@
...
-fi;
-
-if [[ $EXIT_BEHAT_PAYPAL_TEST_CODE -eq 1 && $TRAVIS_PULL_REQUEST = false && ! -z "$SYMFONY__PAYPAL__EXPRESS_CHECKOUT__USERNAME" ]]; then
-    bin/behat --strict -p cached --rerun --tags "@paypal && ~@todo"; EXIT_BEHAT_PAYPAL_TEST_CODE=$?
+if [[ $EXIT_CODE -eq 1 ]]; then
+    bin/behat --strict -f progress -p cached --rerun; EXIT_CODE=$?
 fi;
 
-checkBuild $EXIT_BEHAT_TEST_CODE $EXIT_BEHAT_PAYPAL_TEST_CODE
+exit $EXIT_CODE
```
Comment (lchrusciel) [inline etc/travis/run-behat]: Missing blank line

File: app/config/config_test.yml [hunk 2]
```diff
@@ -53,3 +52,12 @@ monolog:
 
 twig:
     exception_controller: 'Sylius\Bundle\CoreBundle\Controller\VerboseExceptionController::showAction'
+
+payum:
+    gateways:
+        paypal_express_checkout:
+            paypal_express_checkout_nvp:
+                username: EDITME
```
Comment (pjedrzejewski) [inline app/config/config_test.yml]: Maybe we should use "TEST".

**Comment by pjedrzejewski** [comment]
Thanks Arek, nice work!

**Comment by pamil** [comment]
:+1:

**Pull request merged**